program/src/oracle/oracle.c

```diff
@@ -510,6 +510,9 @@ static uint64_t upd_price( SolParameters *prm, SolAccountInfo *ka )
     fptr->conf_     = cptr->conf_;
     fptr->status_   = status;
     fptr->pub_slot_ = cptr->pub_slot_;
+    sol_log("Instruction: UpdatePrice");
+    sol_log("price, conf, status, pub_slot");
```

Comment:
I'm planning to parse the logs and look for `Instruction: UpdatePrice` for all price update txs and the reason why this format is chosen is because other major protocol like Mango and Drift also does this, happy to consider other alternatives, as for `price, conf, status, pub_slot` is to make it clear that the values below is of such order, we could of course remove it and just document it somewhere in our gitbook but I thought this will improve the UX for other protocols especially if they wish to compose on us or build metrics/analytics using our data